apps/table.html

```diff
@@ -215,10 +215,10 @@ <h5 class="modal-title" id="title-of-dialog">Upload New Slide</h5>
 
                                 <h3>Steps for uploading.</h3>
                                 <ul>
-                                    <li>Select the file as Input. The <b>Token</b> field should be filled automatically, and start uploading.</li>
-                                    <li> Provide the destination filename with extension.</li>
-                                    <li>Click on the <b><i class="fas fa-arrow-right"></i></b> button to get check a preview of the slide  </li>
-                                    <li>Finally <b>Finish Upload</b> to post the slide to caMicroscope</li>
+                                    <li>Select the file(.svs) you wish to upload. The <b>Token</b> field should be automatically populated .</li>
```

Comment:
Files don't have to be .svs. 
```suggestion
                                    <li>Select the file you wish to upload. The <b>Token</b> field should be automatically populated .</li>
```